lang/ru.po

```diff
@@ -85,7 +85,7 @@ msgstr "Отмена"
 
 #: apps/postybirb-ui/src/components/form/user-specified-website-options-modal/user-specified-website-options-modal.tsx:36
 msgid "Choose Fields"
-msgstr "Выберите поля"
```

Comment:
Any idea why these strings are being emptied out?

Reply:
Must be some problem with the base switching. I'll fix that rn!